can_ports: ["can0", "can1"]
max_current_A: 2.0
has_endstop: true
move_to_position_tolerance_rad: 0.05
calibration:
    endstop_search_torques_Nm: 
        - 0.22
        - 0.22
        - -0.18
    move_steps: 500
safety_kd:
    - 0.09
    - 0.09
    - 0.05
position_control_gains:
    kp: [7, 7, 6.5]
    kd: [0.01, 0.015, 0.02]

hard_position_limits_lower:
    - -2.0
    - -1.4
    - -3.2
hard_position_limits_upper:
    - 1.6
    - 1.8
    - 3.2

soft_position_limits_lower:
    - -1.6
    - -1.2
    - -3.0
soft_position_limits_upper:
    - 1.0
    - 1.6
    - 3.0

# Set zero-position according to URDF (finger pointing straight down).
home_offset_rad:
    - -1.543
    - -0.987
    - 3.0

# Set the initial position such that it is still save when the stage is at the
# highest level.
initial_position_rad:
    - 0
    - 0.9
    - -1.7

